tests/expr_and_series/str/zfill_test.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import pytest
+
+import narwhals as nw
+from tests.utils import Constructor, ConstructorEager, assert_equal_data
+
+data = {"a": ["-1", "+1", "1", "12", "123", "99999", "+9999", None]}
+polars_expected = {"a": ["-01", "0+1", "001", "012", "123", "99999", "+9999", None]}
+pandas_expected = {"a": ["-01", "+01", "001", "012", "123", "99999", "+9999", None]}
+
+
+def skip_pandas_pyarrow(constructor: Constructor | ConstructorEager) -> None:
+    name: str = constructor.__name__
+    if name == "pandas_pyarrow_constructor":
+        # Skip this test for pandas with pyarrow, as it doesn't support str.zfill
+        raise pytest.skip()
```

Comment:
```suggestion
    if name == "pandas_pyarrow_constructor":
        reason = (
            "pandas with pyarrow backend doesn't support str.zfill, see "
            "https://github.com/pandas-dev/pandas/issues/61485"
        )
        raise pytest.skip(reason=reason)
```

Reply:
Think a second longer on this one, I think we should explicitly xfail (not xskip) it, so that eventually in pandas 3.x it would be fixed and we will automatically spot it